src/components/Categories/Categories.js

```diff
@@ -1,9 +1,7 @@
-import React from 'react'
+import React from 'react';
 
-const Categories = () => {
-  return (
-    <div>Categories</div>
-  )
-}
+const Categories = () => (
+  <div>Categories</div>
+);
```

Comment:
- [ ] The Categories component should have a button with the text 'check status' as mentioned in the [project guidelines. ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md)
![image](https://user-images.githubusercontent.com/58504655/167267126-3a9fdfab-bdc5-4457-9555-b66e81786119.png)